src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (typeof num === 'number') {
```

Comment:
You do not need to check if `num` is a number according to the task instructions. It's safe to assume that the input will always be a number or undefined.